dashboard/app/dsl/external_dsl.rb

```diff
@@ -0,0 +1,17 @@
+class ExternalDSL < ContentDSL
+  def initialize
+    @hash = {href: '', options: {}}
+  end
+
+  def href(url)
+    @hash[:href] = url
+  end
+
+  def method_missing(key, *args)
+    @hash[:options][key.to_sym] = args.first
+  end
```

Comment:
As we discussed yesterday, it would be cleaner to move these options values into an explicit `options` block to give a bit more structure to it, so that typos and potential naming conflicts would be less problematic. But this is fine for now - I might do that as a next step later as I look into adding Markdown support into this level type.